apps/contract-sync-job/src/db-helpers.ts

```diff
@@ -0,0 +1,73 @@
+import { GovrnProtocol } from '@govrn/protocol-client';
+
+export const getOrInsertUser = async (
+  govrn: GovrnProtocol,
+  data: { address: string },
+): Promise<number> => {
+  // Query existing user.
+  const user = await govrn.user.list({
```

Comment:
Could we use get instead of list?

Reply:
`govrn.user.get` uses only `id` number to query.
In this case, we want to query by unique address.